satellite/marketing/marketingweb/server.go

```diff
@@ -83,13 +76,27 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		filepath.Join(s.templateDir, "foModal.html"),
 	)
 
-	home, err := template.New("landingPage").ParseFiles(files...)
+	s.templates, err = template.New("landingPage").ParseFiles(files...)
```

Comment:
This should be propagated up, if this fails here serving in ServeHTTP will panic.

Reply:
changed it to return the error